demo/public/index.html

```diff
@@ -1,11 +1,27 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <!-- Include gtag... data layer commands conflict with measure though -->
```

Comment:
I think this is fine.  If anything, it shows that measurement library can send approximately the same events to GA as gtag would.

Reply:
Hmm, I think that you are right and it might work with the same data layer. Though I already made them separate and I think separate is a bit better, so I won't change it back. I think separate is better because 1) it doesn't log errors to the console like below and 2) we don't send duplicate events. Technically we could just send 1 message to the data layer for each event instead of 1 per library but I feel like that could be limiting if measurement library changes to become less like gtag in the future

![image](https://user-images.githubusercontent.com/19393086/90645956-98d40600-e204-11ea-9bba-d1087eb097d7.png)

Reply:
Yeah, I'm fine leaving them in a separate data layer mostly due to the different config command formats (although it'd also be cool to see them process the same data layer into similar events).  Was just noting that the comment was a bit misleading :).